src/main/java/ru/serce/jnrfuse/AbstractFuseFS.java

```diff
@@ -62,7 +62,11 @@ public AbstractFuseFS() {
                 }
                 break;
             case WINDOWS:
-                String winFspPath = WinPathUtils.getWinFspPath();
+        		if (!System.getProperty("file.encoding").equals("UTF-8"))
+        		{	
+        			System.out.println("UTF-8 encoding required! Current encoding: " + System.getProperty("file.encoding"));
```

Comment:
I'm not sure if it's a good idea to print such errors into stdout. That way, we might mess with the stdout of an application which other apps might rely upon.

Reply:
changed it to 
```
    if (!System.getProperty("file.encoding").equals("UTF-8"))
    {	
         throw new FuseException("UTF-8 encoding required! Current encoding: " + System.getProperty("file.encoding"));
     }
```

Guess that's the better solution. It stops immediately. 

Not sure right now, do I have to close this PR and open a new one or are you able to access the new version?